tests/plugins/test_koji_import.py

```diff
@@ -622,6 +622,46 @@ def test_koji_import_wrong_source_type(self, tmpdir, os_env):
             runner.run()
         assert "plugin 'koji_import' raised an exception: RuntimeError" in str(exc)
 
+    @pytest.mark.parametrize(('isolated'), [
```

Comment:
Maybe also handle `None`, in which case the `'isolated'` label is not set at all?

Reply:
If it's None, should we assume false?

Reply:
Yes. Similarly to `autorebuild`, it's `False` unless proven otherwise. But it's always there.

Reply:
added.